starters/tailwind/src/Breadcrumbs.tsx

```diff
@@ -10,7 +10,7 @@ export function Breadcrumbs<T extends object>(props: BreadcrumbsProps<T>) {
 
 export function Breadcrumb(props: BreadcrumbProps & LinkProps) {
   return (
-    <AriaBreadcrumb {...props} className={twMerge('flex items-center gap-1', props.className)}>
+    <AriaBreadcrumb {...props} className={(renderProps) => twMerge('flex items-center gap-1', typeof props.className === 'function' ? props.className(renderProps) : props.className)}>
```

Comment:
Probably could use `composeTailwindRenderProps` here?

Reply:
ah good point, had to omit the LinkProps due a clash in the render props definitions. The Breadcrumb should probably accept `linkClassname` instead, what do you think?

Reply:
oh hmm, maybe. Someone could easily add that if they needed it. It's just a starter.